meta-mender-tegra/README.md

```diff
@@ -17,7 +17,7 @@ Download the source:
     $ repo init \
             -u https://github.com/mendersoftware/meta-mender-community \
             -m meta-mender-tegra/scripts/manifest-tegra.xml \
-            -b thud
+            -b warrior
```

Comment:
Can you also update the README structure to match the suggested structure that we discussed. You can use [this](https://github.com/mendersoftware/meta-mender-community/blob/warrior/meta-mender-raspberrypi/README.md) as a template

Reply:
Yes, sorry I missed that.